ensime_shared/ensime.py

```diff
@@ -202,9 +210,9 @@ def setup(self, quiet=False, bootstrap_server=False):
         def lazy_initialize_ensime():
             if not self.ensime:
                 called_by = inspect.stack()[4][3]
-                self.log(str(inspect.stack()))
-                self.log("setup(quiet={}, bootstrap_server={}) called by {}()"
-                         .format(quiet, bootstrap_server, called_by))
+                self.log.debug(str(inspect.stack()))
```

Comment:
True, I hadn't really looked at this one yet.

I think the entire fact that this logging statement exists is a smell, related to #325—trying to trace how the launcher actually runs is really convoluted, which I guess is why this is here. There may be some good reason for the way it's done but I don't understand what it is yet. I'd like to push it up to happening from the `Ensime` class without `EnsimeClient` ever touching the launcher directly, but of course that's some separate work.

Is it worth doing anything with this right now?